spec/spec_helper.rb

```diff
@@ -1,6 +1,7 @@
 # Coverage must be enabled before the application is loaded.
 if ENV['COVERAGE']
   require 'simplecov'
+  require 'coveralls'
```

Comment:
I assume we want to tell hound that we prefer single quotes unless interpolating, @keathley. Is that correct?